pkg/provider/azure.go

```diff
@@ -417,6 +417,8 @@ type Cloud struct {
 	pipCache azcache.Resource
 	// use LB frontEndIpConfiguration ID as the key and search for PLS attached to the frontEnd
 	plsCache azcache.Resource
+	// a timed cache storing storage account properties to avoid querying storage account frequently
+	StorageAccountCache azcache.Resource
```

Comment:
Is it necessary to make the variable external?

Reply:
it's internal now